dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/CompanyDataControllerTest.kt

```diff
@@ -114,7 +132,7 @@ class CompanyDataControllerTest {
     }
 
     @Test
-    fun `post a dummy company and check if patching alternative names works as expected`() {
+    fun `post a dummy company and check if patching alternative names and contact details works as expected`() {
```

Comment:
"and contact details" stimmt nicht. Es werden nur alternative names gepatcht.

Reply:
forgotten part of a removed change. will remove it